dashboards/puma/main.json

```diff
@@ -90,8 +90,8 @@
         "display": "line",
         "format": "number",
         "draw_null_as_zero": true,
-        "title": "Puma worker info",
-        "description": "Number of workers per type. Old workers are still running old code after a deploy and should eventually be replaced with fresh workers running new code.",
+        "title": "Workers",
+        "description": "Workers running old code should eventually be replaced with new workers.",
```

Comment:
Not all workers in this graph are 'old' workers. It depends on the value of the 'type' tag.

Reply:
Would it help if I changed it to "Old workers should eventually"...? (to make it clear that I'm referring to those with the `old` type tag)

Reply:
Sure